controllers/userController.js

```diff
@@ -62,6 +63,21 @@ exports.getUsers = async (req, res) => {
 
 exports.updateUser = async (req, res) => {
   try {
+    const idToSearchFor = mongoose.Types.ObjectId(req.user['user:id'])
+    const user = await req.db.User.findOne(idToSearchFor)
+
+    if (!user) {
+      return res.status(HttpStatus.NOT_FOUND).json({
+        success: false,
+        message: 'User not found!'
+      })
+    }
+
+    await req.db.User.updateOne(user, req.body)
+
+    return res.json({
+      success: true
+    })
   } catch (error) {
     req.log.error(`Unable to update user -> ${error}`)
     return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json({
```

Comment:
Here a message please

Reply:
on the 500 response, there should be a message. It wasn't visible at the time of my review, please add a message to the response. Sorry for misunderstanding this